[linux_pam] Failed to build

```
pam_unix_passwd.c:186:16: warning: assignment to ‘struct netconfig *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]
  186 |  while ((nconf = getnetconfig(handle)) != NULL) {
      |                ^
pam_unix_passwd.c:187:21: error: invalid use of undefined type ‘struct netconfig’
  187 |    if (!strcmp(nconf->nc_proto, "udp")) {
      |                     ^~
pam_unix_passwd.c:188:10: warning: implicit declaration of function ‘rpcb_getaddr’ [-Wimplicit-function-declaration]
  188 |      if (rpcb_getaddr(YPPASSWDPROG, YPPASSWDPROC_UPDATE,
      |          ^~~~~~~~~~~~
pam_unix_passwd.c:190:36: warning: passing argument 1 of ‘__taddr2port’ from incompatible pointer type [-Wincompatible-pointer-types]
  190 |               port = __taddr2port (nconf, &svcaddr);
      |                                    ^~~~~
      |                                    |
      |                                    struct netconfig *
pam_unix_passwd.c:111:39: note: expected ‘const struct netconfig *’ but argument is of type ‘struct netconfig *’
  111 | __taddr2port (const struct netconfig *nconf, const struct netbuf *nbuf)
      |               ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~
pam_unix_passwd.c:191:15: warning: implicit declaration of function ‘endnetconfig’ [-Wimplicit-function-declaration]
  191 |               endnetconfig (handle);
      |               ^~~~~~~~~~~~
pam_unix_passwd.c:145:16: warning: unused variable ‘svcaddr’ [-Wunused-variable]
  145 |  struct netbuf svcaddr;
      |                ^~~~~~~
make[3]: *** [Makefile:874: pam_unix_passwd.lo] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/usr/local/tmp/crew/Linux-PAM-1.5.1.tar.xz.dir/Linux-PAM-1.5.1/modules/pam_unix'
make[2]: *** [Makefile:490: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/Linux-PAM-1.5.1.tar.xz.dir/Linux-PAM-1.5.1/modules'
make[1]: *** [Makefile:491: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/Linux-PAM-1.5.1.tar.xz.dir/Linux-PAM-1.5.1'
make: *** [Makefile:421: all] Error 2
linux_pam failed to build: `make V=0 -j4` exited with 2
```

--- a/packages/linux_pam.rb
+++ b/packages/linux_pam.rb
@@ -11,18 +11,20 @@ class Linux_pam < Package
   binary_url ({
     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/linux_pam-1.5.1-chromeos-armv7l.tar.xz',
      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/linux_pam-1.5.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/linux_pam-1.5.1-chromeos-i686.tar.xz',
      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/linux_pam-1.5.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
     aarch64: '50f55a3858081ca4748fba9f731310b81c9ed6ed85597c1fd715a5c2ae1f56d4',
      armv7l: '50f55a3858081ca4748fba9f731310b81c9ed6ed85597c1fd715a5c2ae1f56d4',
+       i686: '5017e846407cb9692a915b1b513616515a45a6143303904125d29f0a43348ab6',
      x86_64: 'af1b7e6212444ba310b0441a5f32697f6478a1b62f79b78153fd49718a4e8465',
   })
 
   depends_on 'libdb' # libdb needs to be built with "--enable-dbm"
   
   def self.build
-    system "./configure #{CREW_OPTIONS} --enable-static"
+    system "./configure #{CREW_OPTIONS} --enable-static --disable-nis"
     system 'make'
   end
 

